examples/differential_rotation/differentially_rotated_coordinate.py

```diff
@@ -0,0 +1,59 @@
+"""
+====================================
+Differentially rotating a coordinate
+====================================
+
+How to differentially rotate a coordinate.
+
+The example uses the `~sunpy.coordinates.metaframes.RotatedSunFrame` coordinate
+metaframe in `sunpy.coordinates` to apply differential rotation to a
+coordinate.  See :ref:`sunpy-coordinates-rotatedsunframe` for more details on
+using `~sunpy.coordinates.metaframes.RotatedSunFrame`.
+"""
+from astropy.coordinates import SkyCoord
+import astropy.units as u
+import matplotlib.pyplot as plt
+import numpy as np
+
+from sunpy.coordinates import RotatedSunFrame
+from sunpy.data.sample import AIA_171_IMAGE
+from sunpy.map import Map
```

Comment:
I am generally not in favour of this import pattern. I think in the rest of the gallery we do `import sunpy.map`? While I appreciate not everyone likes this I think consistency in the gallery is important, and if we want to change it we should do it globally.

Reply:
Changed